### DISCUSSION TITLE
Rogue's Gallery

### DISCUSSION THREAD

**Discussion by MarkJaroski** [jakopako/croncert-config#366] [General]
First post: HTML embeddded in json .... 

why? why? why? why?

https://polesud.ch/em-ajax/get_listings/?per_page=200&orderby=event_start_date&order=ASC&page=1&show_pagination=false&search_location=&search_keywords=

**Comment by jakopako** [comment]
Great idea! :D

**Comment by MarkJaroski** [comment]
https://ebrietas.ch/konzerte

It _looks_ nice but the HTML is all over the place. It was done with some kind of WYSIYG tool, I guess.

**Comment by jakopako** [comment]
https://waxybar.ch/

Nice website. Unfortunately, it's all just images.

**Reply by MarkJaroski** [reply]
Will it upset them if we scrape their Eventfrog site instead?

**Reply by jakopako** [reply]
let's try haha

**Comment by MarkJaroski** [comment]
https://www.24heures.ch/festival-vibiscum-confirme-son-annulation-525568347843

Good thing I didn't invest time in building a scraper config.

**Comment by castaway** [comment]
Hand written WIX pages, the formats for time+date aren't always consistent! https://www.swindonfestivalofliterature.co.uk/monday-5th-may-2025
